### PR TITLE
feature: allow to disable cross zone balancing

### PR DESCRIPTION
**Pull request by szuecs**
https://github.com/zalando-incubator/kube-ingress-aws-controller/compare/v0.14.41...v0.15.5

* feature: allow to disable cross zone balancing
* Limit ingress support to only networking.k8s.io/v1
* fix: Set the default ingress API Version
* updates: aws sdk and uuid dependencies

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/ingress-controller/deployment.yaml [hunk 1]
```diff
@@ -56,7 +56,6 @@ spec:
             - --load-balancer-type={{ .Cluster.ConfigItems.kube_aws_ingress_default_lb_type }}
             # {{ end }}
             - --cert-polling-interval={{ .Cluster.ConfigItems.kube_aws_ingress_controller_cert_polling_interval }}
-            - --ingress-api-version=networking.k8s.io/{{ .Cluster.ConfigItems.kube_aws_ingress_controller_ingress_version }}
```
Comment (AlexanderYastrebov) [inline cluster/manifests/ingress-controller/deployment.yaml:59]: Lets drop kube_aws_ingress_controller_ingress_version from config defaults as well.

Comment (szuecs) [reply cluster/manifests/ingress-controller/deployment.yaml:59]: done

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**